subprojects/packagefiles/cmark-gfm/meson.build

```diff
@@ -94,7 +94,10 @@ src = files(
 )
 
 # Many unused parameters warnings when this warning is enabled.
-add_project_arguments('-Wno-unused-parameter', language: 'c')
+cc = meson.get_compiler('c')
+if (cc.has_argument('-Wno-unused-parameter'))
```

Comment:
`if (expression)` is equivalent to `if (expression)`, the grouping isn't doing anything here.

Also note there is:

```meson
add_project_arguments(cc.get_supported_arguments('-Wno-unused-parameter'), language: 'c')
```

which means you don't need to repeat the argument name.

Reply:
Thanks, the extra brackets were an honest mistake. Incorporated your feedback on the latest commit.